### PR TITLE
make mcq stats labels non varied text

### PR DESCRIPTION
**Pull request by SamamthaH**
image

### CODE REVIEW

**Comment by SamamthaH** [comment]
In this case the only thing in the correct answer and distractor text box was the variable, however it could also look like "the number: {{num}} is even"

File: canvas/services/event.py [hunk 1]
```diff
@@ -44,8 +45,9 @@ def get_question_stats(question):
 
     answers = {}
     if isinstance(question, MultipleChoiceQuestion):
+        choices = json.loads(question.choices) if type(question.choices) == str else question.choices
```
Comment (keyvankhademi) [inline canvas/services/event.py:48]: In what situation do you get `question.choices` to be str?

Comment (SamamthaH) [reply canvas/services/event.py:48]: I am unsure, I took this line from course/models/models.py get_rendered_choices (line 323 specifically) which the original code called instead. I presume it is in there for a reason so I didn't want to mess with it.

**Review by keyvankhademi — APPROVED** [review]

**Comment by keyvankhademi** [comment]
@SamamthaH Please fix the lint

**Comment by SamamthaH** [comment]
I have no idea what the gunicorn file is but apparently that was the thing that needed reformatting. Feels weird to have it tacked on to this pr though.

**Pull request merged**